Development

jakedavison opened the pull request.

jakedavison committed:
Merge branch 'master' of https://github.com/RUGSoftEng/2017-Cognitive-Sensors into feature-feedback

jakedavison committed:
Merge branch 'development' of https://github.com/RUGSoftEng/2017-Cognitive-Sensors into feature-feedback

jeajones committed:
Fixed graphdata class, added extra columns to database

jakedavison committed:
Merge branch 'feature-graphdisplay' of https://github.com/RUGSoftEng/2017-Cognitive-Sensors into feature-feedback

jakedavison committed:
Readied chart display for real data input.

jeajones committed:
Bug fixes and restructuring

jeajones committed:
More bug fixes for graph data

aukeroorda committed:
The questionID stored is now equal to the questionID without the encoded block. Removed println's

jakedavison committed:
Merge pull request #78 from RUGSoftEng/feature-feedback

Feature feedback

jeajones committed:
Fixed constructor for graphdata

jeajones committed:
Merge branch 'feature-graphdata' of https://github.com/RUGSoftEng/2017-Cognitive-Sensors into feature-graphdata

jakedavison committed:
Initiated link between GraphData and Feedback.

jakedavison committed:
Merge branch 'feature-graphdata' into feature-feedback

jakedavison committed:
Merge pull request #79 from RUGSoftEng/feature-feedback

Initiated link between GraphData and Feedback.

jeajones committed:
Fixed game percentages

aukeroorda committed:
Fixed initial time the number is shown not being set. Refactored some initialization

jakedavison committed:
Finalised link between GraphData and Feedback. Feedback starts on session end.

jakedavison committed:
Fixed Quit button.

jakedavison committed:
Tiny bugfix

jeajones committed:
Fixed the total counter

jakedavison committed:
More chart bugfixes.

jakedavison committed:
Merge branch 'feature-feedback' of https://github.com/RUGSoftEng/2017-Cognitive-Sensors into feature-feedback

jakedavison committed:
APK updated.

Commit:
Merge pull request #82 from RUGSoftEng/feature-graphdata

Feature graphdata

Commit:
Merge branch 'development' into feature-feedback

Commit:
Merge pull request #83 from RUGSoftEng/feature-feedback

Feature feedback

PJUllrich commented on Wander/app/src/main/java/com/teamwan/wander/Feedback.java:
```diff
@@ -138,50 +146,103 @@ private void setTypefaces() {
+    }
+
+    public float fetchBarDataAcc(int task){
+        ArrayList<Float> values;
```
This functionality is duplicate with the previous function. The only difference is that it uses a different set of values

PJUllrich commented on Wander/app/src/main/java/com/teamwan/wander/FeedbackFragmentChart.java:
```diff
@@ -69,11 +79,19 @@ private void initialiseFragment(){
-        if (chartData instanceof LineData) {
-            chart.setData((LineData) chartData);
-            chart.invalidate();
+        if (chartTypeLine) {
```
These 2 if statements have the same conditions and can be combined into 1

ThomasdenH replied:
I don't know if I follow, but if you mean `if (titleString.length()>20)` and `if (chartTypeLine)` I disagree. The titles might be changed or a chart could be added so which titles are long might change.

jakedavison replied:
I believe this is solved, was just a small leftover implementation.

PJUllrich reviewed (approved):
Just some small comments to improve code quality

ThomasdenH committed:
Question order depending on answer.

Closes #72

ThomasdenH committed:
Update server for question order

ThomasdenH committed:
Add portrait lock

Commit:
Merge pull request #85 from RUGSoftEng/feature-question-order

Feature question order

Commit:
Merge pull request #87 from RUGSoftEng/feature-portrait

Add portrait lock

ThomasdenH committed:
Refactor

Commit:
Merge pull request #88 from RUGSoftEng/feature-refactor

Refactor

jakedavison committed:
Line charts reformatted and finalised.

jakedavison committed:
Game now detects idle user, closing game after 10 consective non-clicked digits, without saving data.

aukeroorda committed:
Merge pull request #94 from RUGSoftEng/feature-feedback

Feedback pages updated, in-game idle detection added.

aukeroorda committed:
Fixed a small bug in the feedback fragment, implemented local functionallity for per-question, for each answer defining of on/off-task. updated database, updated gradle build.

Commit:
Fixed Notification Pending Intent Removal

-Made the intent to be removed match the currently queued intents..
-Made a function to test this match
-This function needs to be deactivated before shipping.

Commit:
Merge pull request #96 from RUGSoftEng/feature-graphdata

Pulling to test

ThomasdenH committed:
Replace duplicate function with helper function

Commit:
Merge pull request #100 from RUGSoftEng/feature-duplicatefunction

Replace duplicate function with helper function

jeajones committed:
Updated Questions to send onOffTask info

Commit:
Merge pull request #101 from RUGSoftEng/feature-server

Feature server

aukeroorda committed:
Removed column which tells whether a question defines the persons focus, regarded obsolete. Filtering out negative response times (Occured when you exited the game manually).

jakedavison committed:
Questionnaire feature fully implemented.

jakedavison committed:
Questionnaire feature fully implemented.

jakedavison committed:
Questionnaire feature fully implemented.

jeajones committed:
Fixed the PlayerID to match with the server

jakedavison committed:
Merge pull request #102 from RUGSoftEng/feature-notifications

Feature notifications

jeajones committed:
Merge pull request #103 from RUGSoftEng/feature-graphdata

Removed column which tells whether a question defines the persons foc…

jakedavison committed:
Merge branch 'feature-graphdata' into feature-feedback

jakedavison committed:
Merge pull request #104 from RUGSoftEng/feature-feedback

Feature feedback

jeajones committed:
Added missing brace

aukeroorda committed:
Fixed issue #98.

aukeroorda committed:
Merge branch 'feature-graphdata' of https://github.com/RUGSoftEng/2017-Cognitive-Sensors into feature-graphdata

jeajones committed:
Fixed divide by zero bug when no consent is given

jeajones committed:
Merge branch 'feature-graphdata' of https://github.com/RUGSoftEng/2017-Cognitive-Sensors into feature-graphdata

jeajones committed:
Fixed informed consent boolean

jakedavison committed:
Bar Chart formatting finalised.

jakedavison committed:
Merge branch 'feature-graphdata' of https://github.com/RUGSoftEng/2017-Cognitive-Sensors into feature-graphdata

jakedavison committed:
Small fix

jakedavison committed:
Small fix

jakedavison committed:
Small fix

jeajones committed:
Merge pull request #105 from RUGSoftEng/feature-graphdata

Feature graphdata

jeajones committed:
New APK

jeajones committed:
new APK

jeajones committed:
Merge pull request #106 from RUGSoftEng/newAPK

New apk
